packages/jest-config/src/validConfig.js

```diff
@@ -55,9 +55,13 @@ module.exports = ({
   modulePathIgnorePatterns: ['<rootDir>/build/'],
   modulePaths: ['/shared/vendor/modules'],
   name: 'string',
+  noDefaultReporters: false,
```

Comment:
Ah, if the name of the default reporters is simply "default" (instead of "summary, default") then honestly I think we are fine with "reporters" just overwriting the default:

```js
// Includes the default reporters:
reporters: ["default", "custom-reporter"],
// or excludes them:
reporters: ["custom-reporter"],
// or includes some of them:
reporters: ["summary", "custom-reporter"],
```

This basically would make the default reporter an aggregate of other reporters (jest, summary). We can also change this in the code so that the DefaultReporter wraps the jest (new name for current default reporter?) and summary reporter.